azure-pipelines.yml

```diff
@@ -0,0 +1,68 @@
+name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
+variables:
+  CI_NAME: Azure Pipelines
+  CI_BUILD_ID: $(Build.BuildId)
+  CI_BUILD_URL: "https://dev.azure.com/sunpy/sunkit-dem/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp36-* cp37-* cp38-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: ?????
```

Comment:
I'm not sure what the `?????` here is.

Reply:
because you need to configure it on azure. "sunpy" is the right answer here.